core/profiles.py

```diff
@@ -26,6 +26,22 @@ class couchpotato_meta:
     pretty_name = "CouchPotato"
     baseurl = "/couchpotato"
     systemd = "couchpotato"
+    
+class calibrecs_meta:
+    name = "calibre-cs"
+    Pretty_name "Calibre Content Serever"
```

Comment:
typo

Reply:
```suggestion
    pretty_name = “Calibre Content Server"
```
Gotta double check this